### PR TITLE
Ac/fix puppeteer timeout

### PR DESCRIPTION
**Pull request by Heyrows**
## Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] All new and existing tests are passing.

### CODE REVIEW

File: packages/slice-machine/src/models/slice/reducer.ts [hunk 1]
```diff
@@ -85,7 +85,6 @@ export function reducer(
         return {
           ...prevState,
           screenshotUrls: screenshots,
-          infos: prevState.infos,
```
Comment (Miniplop) [inline packages/slice-machine/src/models/slice/reducer.ts:88]: what is that ?

Comment (Heyrows) [reply packages/slice-machine/src/models/slice/reducer.ts:88]: It's a property that is already passed inside the spread 2 line above :)
I just wanted to simplify it.

Comment (Heyrows) [reply packages/slice-machine/src/models/slice/reducer.ts:88]: the type of it is "componentInfos"

Comment (Miniplop) [reply packages/slice-machine/src/models/slice/reducer.ts:88]: ok, thks for the information

**Review by Miniplop — APPROVED** [review]

**Pull request merged**